src/Components/CreateImageWizardV2/CreateImageWizard.tsx

```diff
@@ -1,7 +1,147 @@
-import React from 'react';
+import React, { useState } from 'react';
+
+import {
+  Button,
+  Wizard,
+  WizardFooterWrapper,
+  WizardStep,
+  useWizardContext,
+} from '@patternfly/react-core';
+import { useNavigate } from 'react-router-dom';
+
+import {
+  EnvironmentStateType,
+  filterEnvironment,
+  hasUserSelectedAtLeastOneEnv,
+  useGetAllowedTargets,
+} from './steps/ImageOutput/Environment';
+import ImageOutputStep from './steps/ImageOutput/ImageOutput';
+import ReviewStep from './steps/Review/ReviewStep';
+
+import { RHEL_9, X86_64 } from '../../constants';
+import './CreateImageWizard.scss';
+import { Distributions } from '../../store/imageBuilderApi';
+import { resolveRelPath } from '../../Utilities/path';
+import { ImageBuilderHeader } from '../sharedComponents/ImageBuilderHeader';
+
+/**
+ * @return true if the array in prevAllowedTargets is equivalent to the array
+ * allowedTargets, false otherwise
+ */
+const isIdenticalToPrev = (
+  prevAllowedTargets: string[],
+  allowedTargets: string[]
+) => {
+  let identicalToPrev = true;
+  if (allowedTargets.length === prevAllowedTargets.length) {
+    allowedTargets.forEach((elem) => {
+      if (!prevAllowedTargets.includes(elem)) {
+        identicalToPrev = false;
+      }
+    });
+  } else {
+    identicalToPrev = false;
+  }
+  return identicalToPrev;
+};
+
+/**
+ * The custom wizard footer is only switching the order of the buttons compared
+ * to the default wizard footer from the PF5 library.
+ */
+const CustomWizardFooter = () => {
+  const { goToNextStep, goToPrevStep, close } = useWizardContext();
+  return (
+    <WizardFooterWrapper>
+      <Button variant="secondary" onClick={goToPrevStep}>
+        Back
+      </Button>
+      <Button variant="primary" onClick={goToNextStep}>
+        Next
+      </Button>
+      <Button variant="link" onClick={close}>
+        Cancel
+      </Button>
+    </WizardFooterWrapper>
+  );
+};
 
 const CreateImageWizard = () => {
-  return <></>;
+  const navigate = useNavigate();
+  // Image output step states
+  const [release, setRelease] = useState<Distributions>(RHEL_9);
+  const [arch, setArch] = useState<'x86_64' | 'aarch64'>(X86_64);
```

Comment:
Would it be possible to use the following with an appropriate import?

```suggestion
  const [arch, setArch] = useState<ArchitectureItem['arch']>(X86_64);
```

It could help in the future. In case there's a new arch added there would be no need to go through the enums and update all of those manually.